toptal/picasso
# feat(Button): add checkbox and radio types

[FX-2121](https://toptal-core.atlassian.net/browse/FX-2121)

### Description

- Added `ButtonInput` as a private base for the checkbox and radio button types
- Added `Button.Checkbox`
- Added `Button.Radio`
- Added `Form.ButtonCheckbox`
- Added `Form.ButtonRadio`
- Added unit and visual tests

### How to test

Check the [deployed demo](https://picasso.toptal.net/FX-2121-add-button-checkbox-and-radio/?path=/story/components-button--button).

### Screenshots

<img width="510" alt="Screen Shot 2021-09-21 at 18 47 57" src="https://user-images.githubusercontent.com/17337276/134203666-d0249547-8a54-4f59-a6b7-434da185e6eb.png">

### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>
